src/test/Home.test.js

```diff
@@ -0,0 +1,16 @@
+import { render } from '@testing-library/react';
+import React from 'react';
+import { screen } from '@testing-library/dom';
+import Home from '../components/Home';
+
+test('Check if id: home-page exist', () => {
+  render(<Home />);
+  const home = document.querySelector('#home-page');
+  expect(home).not.toBeNull();
+});
+
+test('Check if title:Welcome to our page exist', () => {
+  render(<Home />);
+  const title = screen.getByText('Welcome to our page');
+  expect(title).toBeInTheDocument();
```

Comment:
**REQUIRED**
This test throws an error. See this :point_down: 
![Screenshot from 2020-12-14 18-12-18](https://user-images.githubusercontent.com/44072711/102147220-66e05280-3e38-11eb-8cee-465e4a97b4c7.png).
Let's fix this, please.